Gemeinsamer Bibliotheksverbund ISBN.js

```diff
@@ -0,0 +1,238 @@
+				}];
+			}
+			//DDC is not the callNumber in Germany
+			item.callNumber = "";
+			item.complete();
+		});
+		translator.translate();
+
+	});
+}/** BEGIN TEST CASES **/
+var testCases = [
+	{
+		"type": "import",
+		"input": "{ \"ISBN\" : \"978-3-642-00230-4\" }",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Eigentumsverfassung und Finanzkrise",
+				"creators": [
+					{
+						"firstName": "Otto",
+						"lastName": "Depenheuer",
+						"creatorType": "editor"
+					},
+					{
+						"lastName": "Symposion. Deutsche Stiftung Eigentum",
+						"fieldMode": true
+					}
+				],
+				"date": "2009",
+				"ISBN": "9783642002304",
+				"language": "ger",
+				"place": "Berlin ;Heidelberg",
+				"publisher": "Springer",
+				"series": "Bibliothek des Eigentums, Im Auftrag der Deutschen Stiftung Eigentum",
+				"seriesNumber": "7",
+				"attachments": [],
+				"tags": [
+					"Aufsatzsammlung",
+					"Aufsatzsammlung",
+					"Constitutional law",
+					"Constitutional law",
+					"Constitutional law",
+					"Eigentum",
+					"Finanzkrise",
+					"Haftung",
+					"Law",
+					"Law",
+					"Law",
+					"Online-Publikation",
+					"Online-Publikation",
+					"Ordnungspolitik"
+				],
+				"notes": [
+					{
+						"note": "\"Unter dem Thema 'Eigentumsverfassung und Finanzkrise' veranstaltete die Deutsche Stiftung Eigentum am 22. April 2009 in Berlin ein Symposion"
+					},
+					{
+						"note": "Die weltweite Finanzkrise ist Anlass, an Funktion und Wirkweise des privaten Eigentums in einer freiheitlichen Gesellschafts- und Wirtschaftsordnung zu erinnern. Privates Eigentum muss es geben, damit Verantwortung zugerechnet und Haftung realisiert, Gewinn und Verlust einem konkreten Verantwortungsträger persönlich zugerechnet werden können. Die Verletzung dieser konstitutiven Regeln einer auf privatem Eigentum basierenden Wirtschaftsordnung ist wesentlich ursächlich für das eingetretene Desaster auf den Finanzmärkten. Wie alle kulturellen Errungenschaften muss auch die Idee des privaten Eigentums, insbesondere die ihr immanente Bereitschaft zur Übernahme persönlicher Verantwortung des Eigentümers, jeder Generation erneut wieder in Erinnerung gerufen, überzeugend um sie geworben und vor allem vorbildhaft von den Akteuren in Politik und Wirtschaft vorgelebt werden. Nur so kann strukturelles Vertrauen in das Finanzsystem wieder gewonnen werden. Denn in ihrer vertrauensbildenden Kraft liegt die ordnungspolitische Funktion der Gewährleistung privaten Eigentums"
+					}
+				],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "import",
+		"input": "{ \"ISBN\" : \"3-86688-240-8\" }",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Bilinguale Lexik nicht materieller lexikalischer Transfer als Folge der aktuellen russisch-deutschen Zweisprachigkeit",
+				"creators": [
+					{
+						"firstName": "Katrin",
+						"lastName": "Karl",
+						"creatorType": "author"
+					}
+				],
+				"date": "2012",
+				"ISBN": "9783866882416",
```

Comment:
Correct. I'll think of how to best implement this. Because hyphenation
happens right before writing to DB, it would be quite different from our
usual sanitization, but I think the test cases should be pretty close to
what Zotero actually stores in the DB.

On Wed, Feb 25, 2015 at 12:47 AM, Philipp Zumstein <notifications@github.com
> wrote:

> In Gemeinsamer Bibliotheksverbund ISBN.js
> <https://github.com/zotero/translators/pull/847#discussion_r25320921>:
>
> > +	{
> > +		"type": "import",
> > +		"input": "{ \"ISBN\" : \"3-86688-240-8\" }",
> > +		"items": [
> > +			{
> > +				"itemType": "book",
> > +				"title": "Bilinguale Lexik nicht materieller lexikalischer Transfer als Folge der aktuellen russisch-deutschen Zweisprachigkeit",
> > +				"creators": [
> > +					{
> > +						"firstName": "Katrin",
> > +						"lastName": "Karl",
> > +						"creatorType": "author"
> > +					}
> > +				],
> > +				"date": "2012",
> > +				"ISBN": "9783866882416",
>
> @aurimasv <https://github.com/aurimasv> : I tried to update the tests
> with Scaffold, but it seems that the hyphenation is not activated there.
> Correct?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zotero/translators/pull/847/files#r25320921>.
>